### ISSUE TITLE
New lines are being added between comments and logic

### ISSUE THREAD

**Issue by codyhoag** [liferay/liferay-frontend-source-formatter#31]
The `npmRunFormat` Gradle task adds a new line after each JSDoc comment. However, it's customary to have the comment directly above the logic. For example, do this:
```
/**
 * Creates a new Person.
 * @class
 */
function Person() {
}
```
instead of this (which is what the npm formatter is currently enforcing):
```
/**
 * Creates a new Person.
 * @class
 */

function Person() {
}
```
This is also a general rule for our Javadocs too.

The formatter should be updated to allow the comments to be directly above the logic.

**Comment by natecavanaugh** [comment]
Hi Cody,
That seems to actually be in violation of how Brian prefers his comments. We specifically add that new line because he prefers to have the whitespace between the comment and code. I'm not sure how it came to be that the javadocs don't have a newline, but I'll check with him and see what the preferred standard should be :)

**Comment by codyhoag** [comment]
Excellent! Thanks, Nate! 😃

**Comment by natecavanaugh** [comment]
Heya @codyhoag 
This should be fixed as of 2.0.12 and commit 3847a82, but let me know if something isn't working as expected :)

Thanks for bringing this to my attention :)
